classes/music_album.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'item'
+require 'date'
+
+class MusicAlbum < Item
+  attr_reader :id, :on_spotify
+
+  def initialize(publish_date, on_spotify)
+    super(publish_date)
+    @id = Random.rand(1...1000)
```

Comment:
The **id** property is created by the Item class. Hence, I do not think you need to specify it again here. Since the **MusicAlbum** class inherits from the **Item** class, the id property is already available.